src/test/javascript/spec/component/text-submission-assessment/text-feedback-conflicts.component.spec.ts

```diff
@@ -161,60 +145,43 @@ describe('TextFeedbackConflictsComponent', () => {
+            imports: [ArtemisTestModule],
             declarations: [
+                TextFeedbackConflictsHeaderComponent,
                 TextFeedbackConflictsComponent,
-                TextSubmissionAssessmentComponent,
                 TextAssessmentAreaComponent,
                 TextblockAssessmentCardComponent,
-                TextblockFeedbackEditorComponent,
-                ManualTextblockSelectionComponent,
-                TextFeedbackConflictsHeaderComponent,
+                MockComponent(TextblockFeedbackEditorComponent),
+                MockComponent(ManualTextblockSelectionComponent),
                 MockComponent(ScoreDisplayComponent),
-                MockComponent(FaIconComponent),
-                MockComponent(FaLayersComponent),
-                MockTranslateValuesDirective,
-                MockComponent(AssessmentLayoutComponent),
-                MockComponent(AssessmentInstructionsComponent),
-                MockComponent(ResizeableContainerComponent),
-                MockComponent(UnreferencedFeedbackComponent),
                 MockPipe(ArtemisTranslatePipe),
                 MockDirective(TranslateDirective),
-                MockDirective(NgbTooltip),
-                MockComponent(ConfirmIconComponent),
-                MockDirective(NgModel),
-                MockComponent(GradingInstructionLinkIconComponent),
-                MockComponent(AssessmentCorrectionRoundBadgeComponent),
-                MockComponent(ManualTextSelectionComponent),
                 MockComponent(AlertComponent),
             ],
             providers: [
                 { provide: ActivatedRoute, useValue: { snapshot: { paramMap: convertToParamMap({ feedbackId: 1 }) } } },
                 { provide: LocalStorageService, useClass: MockSyncStorage },
                 { provide: SessionStorageService, useClass: MockSyncStorage },
+                MockProvider(Router),
             ],
-        })
-            .overrideModule(ArtemisTestModule, {
-                remove: {
-                    declarations: [MockComponent(FaIconComponent), MockComponent(FaLayersComponent)],
-                    exports: [MockComponent(FaIconComponent), MockComponent(FaLayersComponent)],
-                },
-            })
-            .compileComponents();
+        }).compileComponents();
     });
 
     beforeEach(() => {
-        router = TestBed.inject(Router);
-        jest.spyOn(router, 'getCurrentNavigation').mockReturnValue({ extras: { state: { submission: textSubmission } } } as any);
+        jest.spyOn(TestBed.inject(Router), 'getCurrentNavigation').mockReturnValue({ extras: { state: { submission: textSubmission } } } as any);
         fixture = TestBed.createComponent(TextFeedbackConflictsComponent);
         component = fixture.componentInstance;
         fixture.detectChanges();
     });
 
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
     it('should create', () => {
```

Comment:
This test is pretty pointless and probably should be removed. If the initialization fails the beforeEach will throw an error, but the component will never just be undefined